var/spack/repos/builtin/packages/viennarna/package.py

```diff
@@ -54,6 +55,7 @@ def configure_args(self):
         args = self.enable_or_disable('sse')
         args += self.with_or_without('python')
         args += self.with_or_without('perl')
+        args.append('--without-swig')
```

Comment:
Can you do a `swig` variant (default `false`) and have it conflict with `2.4.3`? So that one can get swig support at least for `2.3.5`!